superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -106,6 +110,61 @@ class DashboardList extends React.PureComponent<Props, State> {
   initialSort = [{ id: 'changed_on_delta_humanized', desc: true }];
 
   columns = [
+    {
+      Cell: ({ row: { original } }: any) => {
+        // TODO: return null if no user is logged in
+        const fetchFaveStar = (id: number) => {
```

Comment:
let's move these function definitions out of this function. Looks like they're pretty similar in both charts and dashboards (only a url change). Can we find a way to reuse this code?

Reply:
Would it make sense to add them to `src/views/CRUD/utils.tsx`?

Reply:
I think that makes sense, at least for now.